<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>eView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="ViewWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>280</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="treeObjects">
        <column>
         <property name="text">
          <string>Object</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Class</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="treeProperty">
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuModels">
     <property name="title">
      <string>Models</string>
     </property>
     <addaction name="actionBounding_Box"/>
     <addaction name="actionCollision_Mesh"/>
    </widget>
    <widget class="QMenu" name="menuTerrain">
     <property name="title">
      <string>Terrain</string>
     </property>
     <addaction name="actionNavigation_Mesh"/>
    </widget>
    <addaction name="menuModels"/>
    <addaction name="menuTerrain"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionGo_to_Region"/>
    <addaction name="actionResolve_Motion"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBounding_Box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bounding Box</string>
   </property>
  </action>
  <action name="actionCollision_Mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Collision Mesh</string>
   </property>
  </action>
  <action name="actionNavigation_Mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation Mesh</string>
   </property>
  </action>
  <action name="actionGo_to_Region">
   <property name="text">
    <string>Go to Region</string>
   </property>
  </action>
  <action name="actionResolve_Motion">
   <property name="text">
    <string>Resolve Motion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
